ui/src/views/People/Add.vue

```diff
@@ -3,9 +3,11 @@
     <v-alert v-model="alert" dismissable type="error">
         {{ error }}
     </v-alert>
-    <v-flex xs6 offset-xs3 v-if="allowedToAccess">
-      <h1>Add a Person</h1>
-      <p>
+    <!--<v-flex xs6 offset-xs3 v-if="allowedToAccess">-->
+      <v-flex :class="gridLayout" v-if="allowedToAccess">
```

Comment:
Is this part of the mobile work? If so, that should be in a different branch. Otherwise this code review will keep snowballing.

Reply:
I will create a branch for this